pkg/hub/store/store.go

```diff
@@ -7,34 +7,48 @@ import (
 	"errors"
 	"fmt"
 
-	"go.uber.org/zap"
-	_ "modernc.org/sqlite"
-
 	applicationv1 "github.com/kobsio/kobs/pkg/kube/apis/application/v1"
 	dashboardv1 "github.com/kobsio/kobs/pkg/kube/apis/dashboard/v1"
 	teamv1 "github.com/kobsio/kobs/pkg/kube/apis/team/v1"
 	userv1 "github.com/kobsio/kobs/pkg/kube/apis/user/v1"
 	"github.com/kobsio/kobs/pkg/log"
+	"github.com/kobsio/kobs/pkg/satellite/plugins/plugin"
+
+	"go.uber.org/zap"
+	_ "modernc.org/sqlite"
 )
 
 const schema string = `
+  CREATE TABLE IF NOT EXISTS plugins (
+    name VARCHAR(255),
+    type VARCHAR(255),
+    satellite VARCHAR(255),
+    description TEXT,
+    address TEXT,
+    PRIMARY KEY (name, type, satellite)
+  );
+
+  CREATE TABLE IF NOT EXISTS clusters (
+    name VARCHAR(255),
+    satellite VARCHAR(255),
+    PRIMARY KEY (name, satellite)
+  );
+
   CREATE TABLE IF NOT EXISTS specs (
     type VARCHAR(255),
-	cluster VARCHAR(255),
-	namespace VARCHAR(255),
-	name VARCHAR(255),
+    cluster VARCHAR(255),
+    namespace VARCHAR(255),
+    name VARCHAR(255),
     satellite VARCHAR(255),
-	spec TEXT,
-	PRIMARY KEY (type, cluster, namespace, name)
+    spec TEXT,
+    PRIMARY KEY (type, cluster, namespace, name, satellite)
   );
 `
 
-type Config struct {
-	DSNUri string `json:"dsnUri"`
-}
-
 // Client is the interface with all the methods to interact with the store.
 type Client interface {
+	SavePlugins(satellite string, plugins []plugin.Instance) error
```

Comment:
`GetPlugins` and `GetClusters` is missing in the interface.

Reply:
I added the missing methods